### PR TITLE
19 daily log r

### PR DESCRIPTION
**Pull request by Richileu-A-Bailey**

### CODE REVIEW

File: stattrack/pubspec.yaml [hunk 1]
```diff
@@ -74,6 +74,7 @@ flutter:
     - assets/icons/envelope-solid.svg
     - assets/icons/google.svg
     - assets/icons/square-facebook.svg
+    - assets/foodstockpic.jpg
```
Comment (falniir) [inline stattrack/pubspec.yaml:77]: The following file should be added to assets/images and not to the root of the directory. This should fix also fix the workflow error.

image


DogSmileShyboosGIF

**Review by falniir — COMMENTED** [review]
https://github.com/Bulk-bros/stattrack/pull/38/files#r995003053

**Comment by falniir** [comment]
FridayPaydayGIF

**Comment by jKm00** [comment]
meal card should be clickable (just print something to the terminal, later it will redirect to a new page)

**Review by jKm00 — CHANGES_REQUESTED** [review]
meal card should be clickable (just print something to the terminal, later it will redirect to a new page)

**Pull request merged**